src/main/resources/views/CommercialVehicles/IVA30.hbs

```diff
@@ -1,131 +1,236 @@
+        </div>
+        <div class="header-right">
+            <img src="{{root}}*/assets/images/dvsa-crest-small.png" alt="DVSA" width="110"/>
+        </div>
+    </div>
+    <div class="section-title-border">
+        <h1> INDIVIDUAL VEHICLE APPROVAL (IVA)</h1>
+        <h1> NOTIFICATION OF REFUSAL TO ISSUE AN INDIVIDUAL APPROVAL CERTIFICATE </h1>
+        <p><b>The Road Vehicles (Approval) Regulations 2020</b></p>
+    </div>
 
-                <ul class="item-list">
-                    <li>Item 3           Fuel tanks/rear protective devices - fuel filler arrangements only;</li>
-                    <li>Item 4           Rear registration plate space - all requirements;</li>
-                    <li>Item 7           Audible warning - operation of audible warning device only;</li>
-                    <li>Item 8           Indirect vision - all except field of view;</li>
-                    <li>Item 9           Braking - warning lamps only;</li>
-                    <li>Item 10         EMC - markings only;</li>
-                    <li>Item 12         Interior fittings - no more than 3 separate examples of failure to comply;</li>
-                    <li>Item 13         Anti-theft &amp; immobiliser - all requirements;</li>
-                    <li>Item 16         Exterior projections - no more than 3 separate examples of failure to comply;</li>
-                    <li>Item 17         Speedometer - illumination only;</li>
-                    <li>Item 18         Plates - all requirements;</li>
-                    <li>Item 20-30    Lighting - all except angle of visibility and position;</li>
-                    <li>Item 31         Seat belts - child restraint warning label only;</li>
-                    <li>Item 33         Controls - no more than 3 separate examples of failure to comply;</li>
-                    <li>Item 34         Defrost/demist - all requirements;</li>
-                    <li>Item 35         Wash/wipe - all requirements;</li>
-                    <li>Item 36         Heating - all requirements;</li>
-                    <li>Item 37         Wheel guards - all requirements;</li>
-                    <li>Item 45         Safety glass - approval marking only;</li>
-                    <li>Item 46         Tyres - approval marking, one example only;</li>
+    <p> AN APPOINTMENT FOR RE-EXAMINATION <b>CANNOT</b> BE MADE ON THE DAY ON WHICH THIS EXAMINATION WAS CARRIED OUT
+        PLEASE CONTACT <b> (0300) 200 1188 </b> TO MAKE AN APPOINTMENT QUOTING YOUR TAS REFERENCE NUMBER </p>
 
-                </ul>
-            </div>
+    <!-- put details table here -->
+    <table class="vehicle-detail-table">
+        <thead>
+            <th scope="row" colspan="2" class="section-title-border">
+                <h1> VEHICLE DETAILS </h1>
+            </th>
+        </thead>
+        <tr class="table-row-cell">
+            <td class="table-row-cell"><b>Vehicle/Trailer NR No: </b> <!--Put data here --> </td>
```

Comment:
Width of boxes around Vehicle/Trailer NR No:, Make/Model, and Body Type appears to be too big in comparison to original cert.